cmd/agent/flags.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"flag"
+	"os"
+	"strconv"
+)
+
+var (
+	endpoint       string
+	pollInterval   int
+	reportInterval int
+)
+
+func parseFlags() {
+
+	flag.StringVar(&endpoint, "a", "localhost:8080", "address and port to send report on server")
+	flag.IntVar(&pollInterval, "p", 2, "the frequency of polling metrics from the runtime package")
+	flag.IntVar(&reportInterval, "r", 10, "the frequency of sending metrics to the server")
+
+	flag.Parse()
+
+	if envEndpoint := os.Getenv("ADDRESS"); envEndpoint != "" {
```

Comment:
[SUGGESTION] If I were you, I would use `Builder` pattern.

Reply:
done